eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -39,5 +39,8 @@
     <!-- Transivite dependency of Microsoft.CodeAnalysis.ExternalAccess.AspNetCore -> Microsoft.CodeAnalysis.Features.
          In product build the dependency is bumped to latest, which also bumps the version of DiaSymReader  -->
     <UsagePattern IdentityGlob="Microsoft.DiaSymReader/*1.4.0*" />
+
+    <!-- Transitive dependency of System.Security.Cryptography.Pkcs.5.0.*. -->
+    <UsagePattern IdentityGlob="System.Formats.Asn1/6.0.0" />
```

Comment:
This will likely cause a prebuilt in the VMR. We had to remove System.Formats.Asn1.6.0.0 from SBRP because it contains a vulnerability. Can you bump the package that's bringing this in or possibly add a direct reference to System.Formats.Asn1.6.0.1?

Reply:
We added it as a prebuilt in `main`, can we not do the same here? https://github.com/dotnet/aspnetcore/pull/56947#issuecomment-2251288462

Reply:
Ah yes, I forgot that we went through this process in main, thanks for reminding me. This should be alright to add to the baseline then.